src/imitation/scripts/eval_policy.py

```diff
@@ -106,14 +109,14 @@ def eval_policy(
             policy = ExplorationWrapper(
                 policy,
                 venv,
-                rng=rng,
+                rng=_rnd,
```

Comment:
I am not sure if I get your question. Sacred generates a new seed for each experiment run. The `_rnd` random number generator is generated from this seed. A new generator is used for every captured function where it is requested to ensure that the order of calling the captured functions does not interfere with their return value. You can read more about this [here](https://sacred.readthedocs.io/en/stable/randomness.html#resilience-to-change)

Reply:
I meant, every time `_rnd` is passed to a function, is a new RandomState object created from the global seed, or is it a shared object across functions? I would bet it's pretty likely the first, but if it's the second we don't want to use this feature because it makes replicability super hard (if you change the order functions are run).